src/use-array/index.spec.tsx

```diff
@@ -0,0 +1,93 @@
+import { act, renderHook } from '@testing-library/react-hooks'
+import useArray from './index'
+
+const init = ['1', 'abc', 1, 34, undefined, null]
+
+it('handle value', () => {
+  const { result } = renderHook(() => useArray(init))
+
+  // remove '1'
+  expect(result.current.state.includes('1')).toBe(true)
+  act(() => {
+    result.current.handleValue('1')
+  })
+  expect(result.current.state.length).toBe(5)
+  ;['abc', 1, 34, undefined, null].map((v) => {
+    expect(result.current.state.includes(v)).toBe(true)
+  })
+  expect(result.current.state.includes('1')).toBe(false)
```

Comment:
`expect(result.current.state).toEqual(['abc', 1, 34, undefined, null])` would be sufficient.